metrics.py

```diff
@@ -0,0 +1,65 @@
+"""Metrics different to sklearn are placed here. 
+Some of the functions are from CAML-MIMIC:
+(https://github.com/jamesmullenbach/caml-mimic/blob/master/evaluation.py)."""
+
+
+import numpy as np
+
+
+def intersect_size(y_true, y_pred, axis):
+    # axis=0 for label-level union (macro). axis=1 for instance-level
+    return np.logical_and(y_pred, y_true).sum(axis=axis).astype(float)
+
+
+def macro_precision(y_true, y_pred):
+    num = intersect_size(y_pred, y_true, 0) / (y_pred.sum(axis=0) + 1e-10)
```

Comment:
The order of inputs is not consistent with the function definition.

```suggestion
    num = intersect_size(y_true, y_pred, 0) / (y_pred.sum(axis=0) + 1e-10)
```